WordPressVIPMinimum/Sniffs/Constants/ConstantStringSniff.php

```diff
@@ -55,7 +55,13 @@ public function process_token( $stackPtr ) {
 			return;
 		}
 
-		$nextToken = $this->phpcsFile->findNext( Tokens::$emptyTokens, $nextToken + 1, null, true, null, true );
+		$nextToken     = $this->phpcsFile->findNext( Tokens::$emptyTokens, $nextToken + 1, null, true, null, true );
+		$nextNextToken = $this->phpcsFile->findNext( Tokens::$emptyTokens, $nextToken + 1, null, true, null, true );
+
+		if ( T_NS_C === $this->tokens[ $nextToken ]['code'] && T_STRING_CONCAT === $this->tokens[ $nextNextToken ]['code'] ) {
+			// Namespacing being used, skip to next.
```

Comment:
Could [`is_token_namespaced()`](https://github.com/WordPress/WordPress-Coding-Standards/blob/06966b26d8deb06a91a5c0f99a04802a6ff5e5bf/WordPress/Sniff.php#L1648-L1671) be used here?

Reply:
I think you're right. I had a quick look to see if there were any other utility functions in WPCS that might be able to check for this, but couldn't spot any.

@jrfnl - one for your new Utils package?

Reply:
Possibly, but to be fair, when `__NAMESPACE__` is used, it doesn't necessarily mean that the code is namespaced. `__NAMESPACE__` can evaluate to an empty namespace.

If you like, you can open an issue already in the [Utils repo](https://github.com/PHPCSStandards/PHPCSUtils). The more detailed the issue, the better.

My initial priority for the Utils repo is to get it set up and release v 1.0 with everything from the refactor PR and a little more.

After that I will look at any open issues, so it may be a little while before I even look at it.